engines/glk/zcode/detection_tables.h

```diff
@@ -93,1108 +104,2715 @@ const PlainGameDescriptor ZCODE_GAME_LIST[] = {
 	{ "zcode", "Unknown Z-code game" },
 
 	// English games
-	{ "404life",           "404 - Life not found" },
```

Comment:
This must be reverted.

Reply:
Got it.
Must this identation be kept only for **// English games**, **// Painfull Little Stupid Games** and **// Converted Scott Adams** games sections?

By the way, I've also sorted some sections (**// French games** for example) while adding new titles. Do I also need to restore the original scheme?

Reply:
If by sorting you mean alphabetical sorting, yes, that is a good thing. However, please just do not alter the whitespaces.

Reply:
Thanks @sev- 

Just added a commit to revert old identation.

BTW I'd like to explain what I have in my mind. 
For my little PI400 I'm building a scummvm libretro installation focused on old adventures (mainly Inform ones).

I'd like to apply zcode documentation updates in detection_tables.h to "renew" other GLK subengines with the same style.

So I kindly ask some of your time for these questions:

- No tab character, only whitespaces. Right?
- Why some sections have different identations? For consistency, I think that the same one should always be used. Do you agree to apply the same used for "//English games"?
- I have documented a lot of competitions entries but each competition usually covers a lot of different GLK subsystem so I've choosen to prepended each entry with a prefix specific to that competition. What do you think?

Example:
ZCODE "Metamorphoses" and TADS "Jarod's Journey" in "The 2000 Annual Interactive Fiction Competition" are documented as

engines/glk/zcode/detection_tables.h
```
// The 2000 Annual Interactive Fiction Competition
{ "if00_metamorphoses", "Metamorphoses" },
```
[...]

engines/glk/tads/detection_tables.h
```
// The 2000 Annual Interactive Fiction Competition
{ "if00_jarodsjourney", "Jarod's Journey" },
```
[...]

Reply:
> * No tab character, only whitespaces. Right?

If you're talking about the alignment space for vertical columns, then both tabs are spaces are allowed. Tabs are 4 spaces wide. If you're talking about the indentation, then it is tabs-only.

> * Why some sections have different identations? For consistency, I think that the same one should always be used. Do you agree to apply the same used for "//English games"?

Historical reasons and to make the total width minimal


> * I have documented a lot of competitions entries but each competition usually covers a lot of different GLK subsystem so I've choosen to prepended each entry with a prefix specific to that competition. What do you think?

I like it!

I see no further issues with your changes, merging.